frontend/src/components/stats/CirculatingSupplyStats.tsx

```diff
@@ -24,10 +24,12 @@ const CirculatingSupplyStats = ({ chartStyle }: Props) => {
             circulatingTokensSupply: new BN(circulatingTokensSupply)
               .divn(10 ** 6)
               .divn(10 ** 6)
+              .divn(10 ** 3)
```

Comment:
Why do we have `10^15` here?
It should be `10^24` to convert from yocto to NEAR

Reply:
Because of this https://github.com/near/near-explorer/blob/master/backend/src/db-utils.js#L494-L506
```
SELECT
  DATE_TRUNC('day', TO_TIMESTAMP(DIV(computed_at_block_timestamp, 1000*1000*1000))) AS date,
  DIV(circulating_tokens_supply, 1000*1000*1000) AS circulating_tokens_supply,
  DIV(total_tokens_supply, 1000*1000*1000) as total_tokens_supply
FROM aggregated__circulating_supply
ORDER BY date
```

Reply:
Why did we decide to divide tokens with `10^9` in the select statement?

Reply:
I decide to do so because we do such convertations in other similar requests so from the component side we'll receive the same lenght of values

Reply:
I guess you could be confused with division for timestamps, we usually convert nanos to seconds.
Let's choose one place where we convert yocto to NEAR. I saw such conversions on JS side, so it's better to remove division for tokens in SQL statement

Reply:
Take a look ones again please 😀